src/Models/IoTWorkspaceProject.ts

```diff
@@ -461,10 +463,36 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
       }
     }
 
+    // Configure project and open in container
+    const projectEnvConfiger = new ProjectEnvironmentConfiger();
+    projectEnvConfiger.configureProjectEnvironmentCore(
+        this.extensionContext, this.channel, this.telemetryContext, deviceDir,
+        PlatformType.Arduino, openInNewWindow);

```

Comment:
> PlatformType.Arduino [](start = 7, length = 21)

Is that we always hardcode platform type as "Arduino" here? I think the platform type is retrieved from user selection.

Reply:
Creating an Arduino project equals to create a workspace project, while Embedded Linux project equals to create a contianer project.
During the project creation process, we configure the project for user. User does not need to select platform type.
You can see the platform type selection process in  `ProjectEnvironmnetConfiger.ts` file which is called when user tries to configure a project we don's know(like code cloned from GitHub).